### PR TITLE
Add v2.3.0

### PR DESCRIPTION
**Pull request by empeje**
## Checklist

- [ ] Code is written and works correctly;
- [ ] Changes are covered by tests;
- [ ] Documentation reflects the changes;

### CODE REVIEW

File: 2.3.0/vm.args [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Licensed under the Apache License, Version 2.1 (the "License"); you may not
```
Comment (dogi) [inline 2.3.0/vm.args]: same here

File: 2.3.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# Licensed under the Apache License, Version 2.1 (the "License"); you may not
```
[the remark this answers is not in the document]
Comment (empeje) [reply 2.3.0/Dockerfile]: Agree for the Dockerfile.

File: 2.3.0/vm.args (hunk 1 above)
Comment (empeje) [reply 2.3.0/vm.args]: Agree for the Dockerfile, but here it is a configuration file, I'll just delete the license part.

File: 2.3.0/Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (empeje) [reply 2.3.0/Dockerfile]: Updated, please check.

File: 2.3.0/vm.args (hunk 1 above)
Comment (empeje) [reply 2.3.0/vm.args]: Updated, please check.

**Pull request merged**